tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -219,3 +219,150 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
+                            "d57a45f3763e4827b7c03f03d60dbbe1"],
+                           ['params.`param0` DESC', 'metrics.`metric0` ASC']),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"],
+                           ['params.`param0` DESC', 'params.`param0` ASC'])])
+@pytest.mark.usefixtures('init_store')
+def test__search_runs_multiple_order_by(expected_runs_ids, test_order_by, init_store):
+    actual_runs, next_page_token = init_store._search_runs(experiment_ids=["hjb553MBNoOYfhXjp3Tn"],
+                                                           filter_string='',
+                                                           run_view_type=ViewType.ACTIVE_ONLY,
+                                                           order_by=test_order_by)
+    for i, run in enumerate(actual_runs):
+        assert run._info.run_id == expected_runs_ids[i]
+
+
+@pytest.mark.parametrize("expected_runs_ids,test_view_type",
+                         [(["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"], ViewType.ACTIVE_ONLY),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1", "6bea870aeaad44e487883ece6e16a6da"],
+                             ViewType.ALL),
+                          (["6bea870aeaad44e487883ece6e16a6da"], ViewType.DELETED_ONLY)])
+@pytest.mark.usefixtures('init_store')
+def test__search_runs_run_view_type(expected_runs_ids, test_view_type, init_store):
+    actual_runs, next_page_token = init_store._search_runs(experiment_ids=["hjb553MBNoOYfhXjp3Tn"],
+                                                           filter_string='',
+                                                           run_view_type=test_view_type)
+    for i, run in enumerate(actual_runs):
+        assert run._info.run_id == expected_runs_ids[i]
+
+
+@pytest.mark.parametrize("expected_runs_ids,test_max_results",
+                         [(["1e5200ae248b476cb0e60286e3f061a4"], 1),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"], 10000)])
+@pytest.mark.usefixtures('init_store')
+def test__search_runs_max_results(expected_runs_ids, test_max_results, init_store):
+    actual_runs, next_page_token = init_store._search_runs(experiment_ids=["hjb553MBNoOYfhXjp3Tn"],
+                                                           filter_string='',
+                                                           max_results=test_max_results,
+                                                           run_view_type=ViewType.ACTIVE_ONLY)
+    for i, run in enumerate(actual_runs):
+        assert run._info.run_id == expected_runs_ids[i]
+
```

Comment:
In order to keep a track of the bug of 50000 runs. you can already add a test with 50000 with the `pytest.xfail` annotation: https://docs.pytest.org/en/latest/skipping.html

When this bug will be fixed, the dev will just have to move your test to the runs_max_results one.